sklearn/multiclass.py

```diff
@@ -1085,7 +1091,7 @@ def predict(self, X):
 
         Returns
         -------
-        y : numpy array of shape [n_samples]
+        y : array-like of shape (n_samples,)
```

Comment:
Since we return the array here, it will be for sure a NumPy array, so the docstring should be:

```suggestion
        y : ndarray of shape (n_samples,)
```